seed/seed.json

```diff
@@ -1114,6 +1114,34 @@
                 "channel": ["NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "BraveDebounceStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 5,
+                    "feature_association": {
+                        "enable_feature": ["BraveDebounce"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 95,
+                    "feature_association": {
+                        "disable_feature": ["BraveDebounce"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "94.0.4606.54",
```

Comment:
@pilgrim-brave @moritzhaller this is a wrong version format, see other entries

Reply:
Well I see them. Is there a way to get the version in that format from, say, a Brave Nightly build?

Reply:
it is shown on `brave://system` for example. Or you manually craft it by taking chromium major version (aka `94`) and appending brave version (e.g. `1.31.55`)